src/Files.App/Views/Properties/HashesPage.xaml

```diff
@@ -26,14 +26,69 @@
 
 	<Grid x:Name="RootGrid">
 
+		<!--  Hash Comparison Section  -->
 		<Grid
-			x:Name="HashesListGrid"
+			x:Name="HashComparisonGrid"
 			Margin="12"
 			VerticalAlignment="Top"
 			Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
 			BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
 			BorderThickness="1"
 			CornerRadius="4">
+			<Grid.RowDefinitions>
```

Comment:
I think the row definitions need to be move to `RootGrid`.

Reply:
I resolved this issue by rebasing, please delete your local branch and pull from the remote.

Reply:
Thanks